protocols/credmark/index.json

```diff
@@ -1,6 +1,6 @@
 {
-    "cname": "Credmark",
+    "cname": "credmark",
     "description": "High integrity DeFi data and a risk modeling platform",
     "path": "credmark",
-    "isEnabled": false
+    "isEnabled": false,
```

Comment:
@murphy  Can you please remove the comma after the value 'false' ?

Reply:
or maybe add 
`"type": "snapshot"`
after that

Reply:
Added "type": "snapshot".

🤞🏻 that gets us over the line.